widget/embedded/src/containers/WidgetInfo/WidgetInfo.tsx

```diff
@@ -12,11 +14,26 @@ export const WidgetInfoContext = createContext<
 export function WidgetInfo(props: React.PropsWithChildren) {
   const { manager } = useManager();
   const swaps = getPendingSwaps(manager);
+  const walletsDetails = useWalletsStore.use.connectedWallets();
+  const isWalletLoading = useWalletsStore.use.loading();
+  const totalBalance = calculateWalletUsdValue(walletsDetails);
+  const refetchBalance = useWalletsStore.use.getWalletsDetails();
   const value = useMemo(() => {
     return {
       swaps,
+      walletsDetails,
```

Comment:
I would suggest to expose all of them as a single property. e.g.:

```
swaps,
wallets: {
isLoading,
details,
totalBalance,
refetch
}

```

Reply:
changed.